tooling/cli/src/info/env_rust.rs

```diff
@@ -31,7 +31,7 @@ pub fn items() -> Vec<SectionItem> {
             .unwrap_or_else(|| {
               (
                 format!(
-                  "rustc: {}\nmaybe you don't have rust installed! Visist {}",
+                  "rustc: {}\nMaybe you don't have rust installed! Visist {}",
```

Comment:
wow, I am full of typos
```suggestion
                  "rustc: {}\nMaybe you don't have rust installed! Visit {}",
```

Reply:
Nice catch :D